packages/component/src/Localization/es-ES.json

```diff
@@ -1,33 +1,52 @@
 {
```

Comment:
@compulim @LiweiMa, are we supposed to keep the top comment about file being locked? If so, can we have it added back in to all `.json` files?